README.md

```diff
@@ -1 +1,61 @@
-# Goruchie.github.io
\ No newline at end of file
+# First-Capstone-Project ![](https://img.shields.io/badge/Microverse-blueviolet)
+
+> This is a the first Microverse capstone project for HTML/CSS & Javascript module.
+
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+- Linters
+
+## Live Demo
+
+[Video Demonstration](https://www.loom.com/share/62137c49b6c542c3ad4e83ee94fa49d5)
+
+[Apex Legends Webinar 2022](https://goruchie.github.io/First-Capstone-Project/)
+ 
+## Getting Started
+
+**You can download, take a look and edit this project if you want to, but first make sure that you have the requirements to do it**
+
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Install Git.
+
+### you must have an account on git hub.
+
+### Install Visual Studio Code or any ide that you have.
+
+### Copy this repository link and clone it to your local machine.
+
+
+
+
+
+## Authors
+
+👤 **Marcos Gualtero Lourenzo**
+
+- GitHub: [githubhandle](@https://github.com/Goruchie)
+- Twitter: [@twitterhandle](https://twitter.com/Goruchie2)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/marcos-gualtero-a2aa35246/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] Please 🙏🏽 add the LICENSE file to your repository and make sure the link works as expected.